src/frontend/lib/graphqlQueries.ts

```diff
@@ -51,34 +51,65 @@ export const nodeQueries = {
       }
     }`
   ),
-}
-
-export const strategicPlanQueries = {
   planNodeByAgency: (id: number) => (
-   `query StrategicPlansByAgency {
-  strategicPlansByAgencyGraphql1(filter: {field_agency_target_id: ${id}}) {
-    pageInfo {
-      total
-    }
-    results {
-      ... on NodePlan {
-        id
-        title
-        link {
-          url
-        }
-        goals {
-          ... on NodeGoal {
-            id
-            fieldId
-            title
-            goalType
-            path
+    `query StrategicPlansByAgency {
+   strategicPlansByAgencyGraphql1(filter: {field_agency_target_id: ${id}}) {
+     pageInfo {
+       total
+     }
+     results {
+       ... on NodePlan {
+         id
+         title
+         link {
+           url
+         }
+         goals {
+           ... on NodeGoal {
+             id
+             fieldId
```

Comment:
this field is gone now on goals and breaks the build when in the query

Reply:
Just found this one in my PR of the updated designs. Should be good once I rebase this for the new layout.